spec/views/users/index_html_erb_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Diego', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'I´m a student', posts_counter: 0)
+    @user2 = User.create(name: 'Marcos', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                         bio: 'I´m a student', posts_counter: 1)
+    @user3 = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                         bio: 'I´m a student', posts_counter: 2)
+    visit users_path
+  end
+
+  describe 'users index page' do
+    # Display all the usernames test
+    it 'displays the created users' do
+      expect(page).to have_content('Diego')
+      expect(page).to have_content('Marcos')
+      expect(page).to have_content('Tom')
+    end
+
+    # Profile picture test
+    it 'shows the right photo' do
+      expect(page).to have_xpath("//img[@src='https://unsplash.com/photos/F_-0BxGuVvo']")
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to test if the three photos are existing on the page, I can see that you checking if the picture existed but your tests are checking only one image exist, you're using the same image but if you used different pictures (as in the real scenario) the testes may fail

Reply:
@Ismailco  please I really need help in fixing this error

Reply:
@sonyco-4u, you just need to include three tests instead of one like you did for the user name

Reply:
@Ismailco  Thanks for the corrections, I have added 3 different images and 3 tests for the users